contracts/ModuleRegistry.sol

```diff
@@ -258,9 +211,8 @@ contract ModuleRegistry is IModuleRegistry, EternalStorage {
     function unverifyModule(address _moduleFactory) external {
         // Can be called by the registry owner, the module factory, or the module factory owner
         bool isOwner = msg.sender == owner();
-        bool isFactoryOwner = msg.sender == IOwnable(_moduleFactory).owner();
         bool isFactory = msg.sender == _moduleFactory;
-        require(isOwner || isFactoryOwner || isFactory, "Not authorised");
+        require(isOwner || isFactory, "Not authorised");
```

Comment:
Maybe we should allow `isFactoryOwner` to unverify their modules as well? Given that we might register and verify custom modules for issuers while keeping them as the actual factory owner.